### DISCUSSION TITLE
CUDA124不适配

### DISCUSSION THREAD

**Discussion by Logan66666** [magicpdf/Magic-PDF#644] [General]
CUDA12.4，只支持torch2.4.0/2.4.1，但是限制地最高版本是2.3.1。因此无法使用CUDA加速，出现报错。
希望大佬能帮忙适配一下。

torch地址：https://[download.pytorch.org/whl/torch/](https://download.pytorch.org/whl/torch/)
![image](https://github.com/user-attachments/assets/799d9261-e461-4c41-bdb2-75a20fa6bb00)

2024-09-22 15:53:49.395 | ERROR    | magic_pdf.tools.cli:parse_doc:96 - Torch not compiled with CUDA enabled

**Comment by myhloli** [comment]
可以用torch2.3.1+cuda12.1

**Comment by Logan66666** [comment]
您好，邮件已收到。谢谢！
